vaadin-markdown-flow-parent/vaadin-markdown-testbench/src/main/java/com/vaadin/flow/component/markdown/testbench/MarkdownElement.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2000-2025 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.markdown.testbench;
+
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.testbench.elementsbase.Element;
+
+/**
+ * A TestBench element representing a <code>&lt;vaadin-markdown&gt;</code>
+ * element.
+ */
+@Element("vaadin-markdown")
+public class MarkdownElement extends TestBenchElement {
+
+    /**
+     * Gets the markdown content of the element.
```

Comment:
Could use some additional comment that rendered content can be queried using regular testbench APIs.

Reply:
Done